api/programmatic-types.graphql

```diff
@@ -84,7 +84,6 @@ type PoolCandidateSearchRequestPaginator {
 "Allowed column names for Query.poolCandidatesPaginated.orderBy."
 enum QueryPoolCandidatesPaginatedOrderByUserColumn {
   PRIORITY_WEIGHT
-  JOB_LOOKING_STATUS
```

Comment:
Huh, not related to this PR but that definitely should have been gone already.  :shrug:

Reply:
Ha, yeah. I was a little confused by this too but realized it was not actually supposed to be there so left it 😅